openapi2jsonschema/util.py

```diff
@@ -111,3 +113,49 @@ def append_no_duplicates(obj, key, value):
         obj[key] = []
     if value not in obj[key]:
         obj[key].append(value)
+
+
+def get_components_from_body_definition(body_definition, prefix=""):
+    MIMETYPE_TO_TYPENAME_MAP = {
+        "application/json": "json",
+        "application/vnd.api+json": "jsonapi",
+    }
+    result = {}
+    for mimetype, definition in body_definition["content"].items():
```

Comment:
```suggestion
    for mimetype, definition in body_definition.get("content", {}).items():
```

Reply:
Responses don't necessarily have a content section.

Reply:
Applied the suggestion, thanks!